api/envoy/extensions/filters/http/language/v3/language.proto

```diff
@@ -0,0 +1,44 @@
+syntax = "proto3";
+
+package envoy.extensions.filters.http.language.v3;
+
+import "udpa/annotations/status.proto";
+import "validate/validate.proto";
+
+option java_package = "io.envoyproxy.envoy.extensions.filters.http.language.v3";
+option java_outer_classname = "LanguageProto";
+option java_multiple_files = true;
+option go_package = "github.com/envoyproxy/go-control-plane/envoy/extensions/filters/http/language/v3;languagev3";
+option (udpa.annotations.file_status).package_version_status = ACTIVE;
+
+// [#protodoc-title: Language]
+// Language :ref:`configuration overview <config_http_filters_language>`.
+// [#extension: envoy.filters.http.language]
+
+// Language detection filter config.
+message Language {
+  // The default language.
+  //
+  // Unicode locale identifier string format
+  // See https://unicode-org.github.io/icu/userguide/locale/
+  string default_language = 1 [(validate.rules).string = {min_len: 2}];
+
+  // The list of supported languages.
+  //
+  // Unicode locale identifier string format
+  // See https://unicode-org.github.io/icu/userguide/locale/
+  repeated string supported_languages = 2
+      [(validate.rules).repeated = {items {string {min_len: 2}}}];
```

Comment:
Must there be at least one supported language? if so, consider adding `min_items` constraint.

Reply:
Updated the validator

```proto
repeated string supported_languages = 2 [(validate.rules).repeated = {
  min_items: 1
  unique: true
  items {string {min_len: 2}}
}];
```